_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,465 @@
+            </label>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6 example-set-hd">Checkboxes</h4>
+<div class="example-set">
+    <fieldset class="field-group">
+        <legend class="field-group-hd field-label">Select your favorite pizza toppings (default checkboxes)</legend>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_01">
+                <input type="checkbox" class="field-input input-checkbox" name="checkboxes_01" id="checkbox_01" value="Pepperoni" checked="checked">
+                <span class="field-input-label">Pepperoni</span>
+            </label>
+        </div>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_02">
+                <input type="checkbox" class="field-input input-checkbox" name="checkboxes_01" id="checkbox_02" value="Sausage">
+                <span class="field-input-label">Sausage</span>
+            </label>
+        </div>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_03">
+                <input type="checkbox" class="field-input input-checkbox" name="checkboxes_01" id="checkbox_03" value="Cheese" checked="checked">
+                <span class="field-input-label">Cheese</span>
+            </label>
+        </div>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_04">
+                <input type="checkbox" class="field-input input-checkbox" name="checkboxes_01" id="checkbox_04" value="Sauce" disabled>
+                <span class="field-input-label">Sauce (disabled checkbox)</span>
+            </label>
+        </div>
+    </fieldset>
+    <fieldset class="field-group">
+        <legend class="field-group-hd field-label">What is your favorite cereal? (enhanced checkboxes)</legend>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_05">
+                <input type="checkbox" class="field-input input-checkbox replace-checkbox" name="checkboxes_02" id="checkbox_05" value="Cinnamon Toast Crunch" checked="checked">
+                <span class="field-input-label">Cinnamon Toast Crunch</span>
+            </label>
+        </div>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_06">
+                <input type="checkbox" class="field-input input-checkbox replace-checkbox" name="checkboxes_02" id="checkbox_06" value="Peanut Butter Cap'n Crunch">
+                <span class="field-input-label">Peanut Butter Cap'n Crunch</span>
+            </label>
+        </div>
+        <div class="field">
+            <label class="field-label label-inline" for="checkbox_07">
+                <input type="checkbox" class="field-input input-checkbox replace-checkbox" name="checkboxes_02" id="checkbox_07" value="Cookie Crisp" checked="checked">
```

Comment:
NIT: Let's keep this consistent with our attribute standards (and what you're doing with required fields/attrs) by just using the ``checked`` notation here. Cool?